src/sass/modules/_m-form-elements.scss

```diff
@@ -75,9 +75,9 @@ button.form-button-disabled {
   // See 18F/web-design-standards#1490
   // This fixes things in the meantime.
   [type="checkbox"] {
-    height: auto !important;
+    height: 1.8rem !important;
```

Comment:
I can't currently get into prescriptions or messaging to test (I am logged in, though). Can anybody who can get it test just make sure it doesn't mess up the styling in there? Again, I think it's probably alright, but I would like to just make sure.